### ISSUE TITLE
Can't play emulator and record.py together

### ISSUE THREAD

**Issue by shelpert** [kevinhughes27/TensorKart#71]
Hi, I've been trying to configure a Switch Pro Controller to work with this project, so I'm using `pygame_sdl2.controller` instead of `inputs`. I got the Python UI working, but when I run `record.py` after running the emulator, the emulator can't detect the controller anymore. I'm wondering if you encountered this problem at all, or if it has to do with the change in packages.

**Comment by kevinhughes27** [comment]
Does the emulator work with the controller without running `record.py`? I imagine it is possible that the `pygame_sdl2.controller` takes over the controller exclusively which could be your issue but that is just speculation. If you figure it out please report back!

**Comment by shelpert** [comment]
Yes, it works until I run `record.py` and then again after I quit it. I haven't found a solution yet but I'm working on it! What kind of OS did you originally run this on, Mac, Windows or Linux?

**Comment by kevinhughes27** [comment]
Sounds like it is taking over the input exclusively then. I ran it on Linux.
